### PR TITLE
X509 genkey

### PR DESCRIPTION
**Pull request by angolini**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 40011e8530a8301a55418e79faf671517b7e6f7c are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1100/docs/artifacts/html/index.html

**Review by vanmaegima — APPROVED** [review]
lgtm

File: source/reference-manual/factory/factory-keys.rst [hunk 1]
```diff
@@ -116,16 +118,53 @@ OP-TEE keys
...
 
         Some already created FoundriesFactories does not have the ``factory-keys``
         directory with the set of keys and certificates. In this case, the commands
-        can be used to create the files.
\ No newline at end of file
+        can be used to create the files.
+
+.. tip::
+        Don't forget to push the new keys to get it included in the next CI
+        build
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-keys.rst:168]: build.

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Review by MrCry0 — APPROVED** [review]
With Vanessa's note - LGTM.

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Comment by doanac** [comment]
Docs for f25484db0186f9b9b1530e7a211d7d809feef0c9 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1106/docs/artifacts/html/index.html

**Pull request merged**